docs/book/src/appendix/keywords.md

```diff
@@ -0,0 +1,89 @@
+# [Appendix A: Keywords](#appendix-a-keywords)
+
+The following list contains keywords that are reserved for current or
+future use by the Sway language. As such, they cannot be used as
+identifiers. Identifiers are names of functions, variables,
+parameters, modules, constants, attributes, types or
+traits, ect.
+
+## [Keywords Currently in Use](#keywords-currently-in-use)
+
+The following is a list of keywords currently in use, with their
+functionality described.
+
+- `as` - perform primitive casting, or rename items in `use` statements
+- [`abi`](https://fuellabs.github.io/sway/master/book/sway-program-types/smart_contracts.html#the-abi-declaration) - defines a smart contract ABI in a syntactcally similar way to traits
+- [`break`](https://fuellabs.github.io/sway/v0.44.0/book/basics/control_flow.html#break-and-continue) - exit a loop immediately
+- [`const`](https://fuellabs.github.io/sway/v0.44.0/book/basics/constants.html) - define constant items
+- [`continue`](https://fuellabs.github.io/sway/v0.44.0/book/basics/control_flow.html#break-and-continue) - continue to the next loop iteration
+- `dep` - call dependencies outside of the current `Forc.toml` scope
```

Comment:
Afaik `dep` has been deprecated in favour of `mod`. There is only 1 place we use it in the repo which is surprising because I'm almost certain that no longer works. I believe that 5 month old file has been forgotten.

I'll double check internally but I'm pretty sure this should be removed.